src/fiskaltrust.ifPOS/v2/me/Models/ApprovalType.cs

```diff
@@ -0,0 +1,14 @@
+using System.Runtime.Serialization;
+namespace fiskaltrust.ifPOS.v2.me
+{
+    [DataContract]
+    public class ApprovalType
+    {
+        public decimal? DiscountAmt { get; set; }
```

Comment:
I think we'll need to add the `[DataMember(Order = xxx)]` attribute to all of these properties to make it work with grpc-net :(

Reply:
Done 😃